ios/Classes/PspdfkitPlugin.m

```diff
@@ -41,23 +46,113 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     }
 }
 
-- (PSPDFDocument *)PSPDFDocument:(NSString *)string {
+# pragma mark - Private methods
+
+- (PSPDFDocument *)PSPDFDocument:(NSString *)path {
     NSURL *url;
     
-    if ([string hasPrefix:@"/"]) {
-        url = [NSURL fileURLWithPath:string];
+    if ([path hasPrefix:@"/"]) {
+        url = [NSURL fileURLWithPath:path];
     } else {
-        url = [NSBundle.mainBundle URLForResource:string withExtension:nil];
+        url = [NSBundle.mainBundle URLForResource:path withExtension:nil];
     }
-
-    NSString *fileExtension = url.pathExtension.lowercaseString;
-    BOOL isImageFile = [fileExtension isEqualToString:@"png"] || [fileExtension isEqualToString:@"jpeg"] || [fileExtension isEqualToString:@"jpg"];
-    if (isImageFile) {
+    
+    if ([self isImageDocument:path]) {
         return [[PSPDFImageDocument alloc] initWithImageURL:url];
     } else {
         return [[PSPDFDocument alloc] initWithURL:url];
     }
 }
 
+- (PSPDFConfiguration *)PSPDFConfiguration:(NSDictionary *)dictionary isImageDocument:(BOOL)isImageDocument {
```

Comment:
Is there any specific reason most of the methods here have a `PSPDF` prefix? I think it would be more readable, and also more conventional to omit it. What do you think?

Reply:
Good point! `PSPDFDocument` was already there, so I wanted to be consistent, but I agree that it looks weird. I'll remove the prefix.